lte/gateway/c/oai/tasks/mme_app/mme_app_state_converter.cpp

```diff
@@ -705,6 +705,9 @@ void MmeNasStateConverter::proto_to_ue_mm_context(
   proto_to_mme_app_timer(
       ue_context_proto.ue_context_modification_timer(),
       &state_ue_mm_context->ue_context_modification_timer);
+  proto_to_mme_app_timer(
```

Comment:
@Vyas20 , after discussion with @ulaskozat created https://github.com/magma/magma/pull/5637 to separate this commit, so that it can be backported to v1.4 release branch.